<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSWidget</class>
 <widget class="QWidget" name="BCSWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="50,550">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleWithReturnWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{
background-color: rgb(81, 80, 80);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <widget class="QPushButton" name="bcsManageButton">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>24</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-family: &quot;微软雅黑&quot;;
	font-size: 14px;
	color: #333333;
	background: #FFFFFF;

	border: 1px solid #363636;
	border-bottom:0px solid #363636;
	border-top-left-radius:8px;
	border-top-right-radius:8px;
	border-bottom-right-radius:0px;
	border-bottom-left-radius:0px;
}
</string>
         </property>
         <property name="text">
          <string>条码秤管理</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bcsGoodsGroupButton">
         <property name="geometry">
          <rect>
           <x>103</x>
           <y>24</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-family: &quot;微软雅黑&quot;;
	font-size: 14px;
	color: #FFFFFF;
	background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 #5D5D5D, stop:1 #595959);
	border: 1px solid #363636;
	border-bottom:0px solid #363636;
	border-top-left-radius:8px;
	border-top-right-radius:8px;
	border-bottom-right-radius:0px;
	border-bottom-left-radius:0px;
}</string>
         </property>
         <property name="text">
          <string>商品组管理</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bcsSendDownButton">
         <property name="geometry">
          <rect>
           <x>635</x>
           <y>14</y>
           <width>125</width>
           <height>32</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-family: &quot;微软雅黑&quot;;
	font-size: 14px;
	color: #FFFFFF;
	background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 #E89161, stop:1 #CD7646);
	border: 1px solid #363636;
	border-radius: 2px;
}

QPushButton:pressed{
	background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 #CD7646, stop:1 #E89161);
}

</string>
         </property>
         <property name="text">
          <string>下发条码秤</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="BCSManageWidget" name="bcsManagePage"/>
        <widget class="BCSGroupManageWidget" name="bcsGroupManagePage"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWithReturnWidget</class>
   <extends>QWidget</extends>
   <header>TitleWithReturnWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BCSManageWidget</class>
   <extends>QWidget</extends>
   <header>BCSManageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BCSGroupManageWidget</class>
   <extends>QWidget</extends>
   <header>BCSGroupManageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
